sql/catalyst/src/main/scala/org/apache/spark/sql/types/numerics.scala

```diff
@@ -17,9 +17,11 @@
 
 package org.apache.spark.sql.types
 
-import scala.math.Numeric.{ByteIsIntegral, IntIsIntegral, LongIsIntegral, ShortIsIntegral}
+import scala.math.Numeric._
 import scala.math.Ordering
 
+import org.apache.spark.sql.types.Decimal.DecimalIsConflicted
```

Comment:
nit: I am wondering about moving it here...

Reply:
Do you mean moving `DecimalIsConflicted` into `numerics.scala`? I think it is fine keeping it in `Decimal.scala` for now.

Reply:
Yes, I meant that. Not a big deal, just to have everything colocated. We can also do in another PR